package.json

```diff
@@ -72,6 +72,9 @@
       "pre-commit": "lint-staged"
     }
   },
+  "dependencies": {
+    "unfetch": "4.1.0"
+  },
```

Comment:
remove this dep from the root `package.json`. 

if you don't want `yarn` to hoist the `unfetch` package out of `driver/node_modules` then add `{ "nohoist": "unfetch" }` option to `packages/driver/package.json`